pkg/scheduler/core/generic_scheduler.go

```diff
@@ -444,7 +444,9 @@ func (g *genericScheduler) findNodesThatFit(pod *v1.Pod, nodes []*v1.Node) ([]*v
 
 		// Create filtered list with enough space to avoid growing it
 		// and allow assigning.
-		filtered = make([]*v1.Node, numNodesToFind)
+		// filtered is not exact numNodesToFind,
+		// need a buffer space for 16 `ParallelizeUntil` goroutine
+		filtered = make([]*v1.Node, numNodesToFind+16)
```

Comment:
please make 15 a constant and use the same thing in all places - otherwise it's too easy to change one thing without changing the other